hugo/layouts/partials/header.html

```diff
@@ -1,49 +1,67 @@
-                                </a>
-                            </li>
-                        {{ end }}
-                    </ul>
-                </li>
-                <li>
-                    <a href="http://talks.sourced.tech" target="_blank" title="Talks">Talks</a>
-                </li>
-                <li>
-                    <a href="https://blog.sourced.tech" title="Blog">Blog</a>
-                </li>
-                <li {{ if eq .URL "stack" }}class="active"{{ end }}>
-                    <a href="{{ "/stack" | absURL }}" title="Our stack">Stack</a>
-                </li>
-                <li {{ if eq .URL "team" }}class="active" {{ end }}>
-                    <a href="{{ "/team" | absURL }}" title="Team">Team</a>
-                </li>
-                <li {{ if eq .URL "careers" }}class="active" {{ end }}>
-                    <a href="{{ "/careers" | absURL }}" title="Our Job Opportunities">Careers</a>
-                </li>
-            </ul>
+                        <ul class="topbar__submenu" id="submenu">
+                            {{ range $k := $ctx.Site.Data.categories.categories.order }}
+                                {{ $section := (index $categories $k) }}
+                                <li {{ if eq $category $k }}class="active"{{end}}>
+                                    <a href="{{ "/projects" | absURL }}#{{ $k }}" title="{{ $section.name }}">
+                                        {{ partial "svg" $k }}
+                                        <span>{{ $section.name }}</span>
+                                    </a>
+                                </li>
+                            {{ end }}
+                        </ul>
+                    </li>
+                    <li>
+                        <a href="http://talks.sourced.tech" target="_blank" title="Talks">Talks</a>
+                    </li>
+                    <li>
+                        <a href="https://blog.sourced.tech" title="Blog">Blog</a>
+                    </li>
+                    <li {{ if eq $ctx.URL "stack" }}class="active"{{ end }}>
+                        <a href="{{ "/stack" | absURL }}" title="Our stack">Stack</a>
+                    </li>
+                    <li {{ if eq $ctx.URL "team" }}class="active" {{ end }}>
+                        <a href="{{ "/team" | absURL }}" title="Team">Team</a>
+                    </li>
+                    <li {{ if eq $ctx.URL "careers" }}class="active" {{ end }}>
+                        <a href="{{ "/careers" | absURL }}" title="Our Job Opportunities">Careers</a>
+                    </li>
+                </ul>
+            </div>
         </div>
-    </div>
-</nav>
+    </nav>
+
+    {{ $secondary_nav := default "" .secondary_nav }}
+    {{ if or (ne $category "") (ne $secondary_nav "") }}
+    <nav class="secondary-topbar">
+        <div>
+            {{ if ne $category "" }}
+                {{ partial "project_nav.html" . }}
```

Comment:
Nope, one is `projects/nav.html`, which is the nav for the projects page itself, and this is `project_nav.html` which is specific for each single project page. Gave it this name for now since we won't know how the hell it's called for now. We can rename it later to what it actually is.